src/common/docker-compose/index.ts

```diff
@@ -13,24 +17,27 @@ export const generate = (dependency_manager: DependencyManager): DockerComposeTe
 
   // Enrich base service details
   dependency_manager.graph.nodes.forEach(node => {
-    compose.services[node.normalized_ref] = {
-      ports: [`${node.ports.expose}:${node.ports.target}`],
-      depends_on: [],
-      environment: {
-        HOST: node.normalized_ref,
-        PORT: node.ports.target.toString(),
-        ARCHITECT: JSON.stringify({
-          [node.name]: {
-            host: `${node.protocol}${node.normalized_ref}`,
-            port: node.ports.target.toString(),
-            datastores: {},
-            subscriptions: {},
-          },
-        }),
-        ARCHITECT_CURRENT_SERVICE: node.name,
-        ...node.parameters,
-      },
-    };
+
+    if (!isLocalDatastore(node)) {
```

Comment:
This would make more sense to generalize the node to be able to assert something like `if (!node.customHost()) { ... }`. That way you can make the solution work when regular services have host overrides too.